addons/web/static/src/core/transition.js

```diff
@@ -30,6 +32,7 @@ export function useTransition({
     initialVisibility = true,
     leaveDuration = 500,
     onLeave = () => {},
+    noTransitionWhenAlreadyMounted = false,
```

Comment:
can't we simplify this name to ``transitionWhenMounted`` ? currently it has no which might result in double negatives and thus confusing logic 🤔

Reply:
Ok